cli/Application/Command/Make/Langtemplates.php

```diff
@@ -73,7 +73,7 @@ public function execute()
 		$this->getApplication()->outputTitle('Make Language templates');
 
 		ExtensionHelper::addDomainPath('Core', JPATH_ROOT . '/src');
-		ExtensionHelper::addDomainPath('CoreJS', JPATH_ROOT . '/www/media');
+		ExtensionHelper::addDomainPath('CoreJS', JPATH_ROOT . '/www/media/js');
 		ExtensionHelper::addDomainPath('Template', JPATH_ROOT . '/cache/twig');
```

Comment:
LOL so I'm using this exact same path on the live server for the live Twig caching.  Is this conflicting in any way when I run the language update commands from there?  (I know, unrelated to this PR, but it kinda caught my eye)